bots/images/scripts/ovirt.setup

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -eux
+
+echo foobar | passwd --stdin root
+
+/usr/bin/systemctl stop firewalld NetworkManager kdump
+/usr/bin/systemctl disable firewalld NetworkManager kdump
+iptables -F
+
+FQDN=ovirt.cockpit.lan
+hostnamectl set-hostname ${FQDN}
+
+# Latest stable repos
+rpm -ivh http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-ovirt-*
+rpm --import https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
```

Comment:
Likewise, this looks too low-level; shouldn't that be done by something like `yum-config-manager --enable epel..`?